tool/gravity/cli/helm.go

```diff
@@ -187,9 +203,10 @@ func releaseInstall(env *localenv.LocalEnvironment, conf releaseInstallConfig) e
 	return nil
 }
 
-func releaseList(env *localenv.LocalEnvironment, all bool) error {
+func releaseList(env *localenv.LocalEnvironment, all bool, tillerNamespace string) error {
 	helmClient, err := helm.NewClient(helm.ClientConfig{
-		DNSAddress: env.DNS.Addr(),
+		DNSAddress:      env.DNS.Addr(),
```

Comment:
Sort of off-topic but this really needs to be hidden behind some sort of address resolver interface with `env` implementing it using the specific DNS configuration - w/o to need to sprinkle the address everywhere.